client/components/establishment-selector.js

```diff
@@ -10,7 +10,25 @@ const revealContent = `To change the primary establishment you must:
 * review any related sections of the application that appear as ‘incomplete'`
 
 export default function EstablishmentSelector({ value, onFieldChange, review, diff, ...props }) {
-  const { establishments, canTransfer, establishment, transferInProgress, readonly, project } = useSelector(state => state.application, shallowEqual);
+  const {
+    establishments,
+    canTransfer,
+    establishment,
+    transferInProgress,
+    readonly,
+    project,
+    isGranted,
+    legacyGranted
```

Comment:
The two "granted" props are the only things that have been added here, everything else is newlines.